dpypelines/pipeline/dataset_ingress_v1.py

```diff
@@ -62,30 +67,33 @@ def dataset_ingress_v1(files_dir: str, pipeline_config: dict):
                 )
                 raise ValueError(msg)
         except Exception as err:
+            logger.error(f"Error while looking for required file {required_file}", 
+            err, 
+            data={"required_file": required_file, 
+            "required_file_patterns": required_file_patterns, 
+            "local_store": local_store})
             de_notifier.failure()
-            raise Exception(
-                message.unexpected_error(
-                    f"Error while looking for required file {required_file}", err
-                )
-            ) from err
+
+            raise err
 
     # Extract the patterns for supplementary distributions from the pipeline configuration
     try:
         supplementary_distribution_patterns = (
             matching.get_supplementary_distribution_patterns(pipeline_config)
         )
+        logger.info("Successfully retrieved supplementary distribution patterns from pipeline config",
+         data={"supplementary_distribution_pattenrs": supplementary_distribution_patterns})
     except Exception as err:
+        logger.error("Failed to get supplementary distribution patterns", err, data={"local_store": local_store})
```

Comment:
pass the config rather than the store please, supplementary distributions are in the config and that'll be the thing we need to know about if we cant get them.